oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -320,9 +320,7 @@ public APIGatewayProxyResponseEvent authoriseRequestHandler(
                         && authRequest.getPrompt().contains(Prompt.Type.LOGIN);
         var identityRequested =
                 identityRequired(
-                        authRequest.toParameters(),
```

Comment:
this is orch, we definitely need to keep this, just putting in a comment here so we don't merge this as-is.